Sources/Error Handling/SKError+Extensions.swift

```diff
@@ -61,6 +61,21 @@ extension SKError {
             return ErrorUtils.storeProblemError(error: self)
 
         @unknown default:
+            switch self.code.rawValue {
+            case 907: // "Unhandled exception"
```

Comment:
Maybe we could create a constant for the magic number so the comment isn't needed? 

i.e.: 
```swift
switch self.code.rawValue {
    case .unhandledException: ...
```

Reply:
Yup great idea